src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        int ball1 = lottery.getRandomBall();
+        int ball2 = lottery.getRandomBall();
+        int ball3 = lottery.getRandomBall();
```

Comment:
Instead of creating separate variables for each ball, consider using an array or collection to store them, especially if the number of balls can change in the future. This will make your code more scalable and easier to manage.